### ISSUE TITLE
Warnings in output

### ISSUE THREAD

**Issue by protbiochem** [BenevolentAI/DeeplyTough#7]
As I'm running the custom evaluation on the sample dataset I get the following warnings at the very end

/uufs/chpc.utah.edu/common/home/u1261874/software/pkg/miniconda3/envs/deeplytough/lib/python3.6/site-packages/torch/utils/checkpoint.py:21: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
/uufs/chpc.utah.edu/common/home/u1261874/software/pkg/miniconda3/envs/deeplytough/lib/python3.6/site-packages/torch/nn/functional.py:1332: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:53<00:00, 53.59s/it]
7it [00:00, 7270.96it/s]

I still get an output, I just want to make sure that this is normal, and not messing up the results I'm getting. If so, what do I need to do to fix it?

Thanks

**Comment by mys007** [comment]
Thanks for your question. These warnings won't affect the results you're getting.

They can very likely be removed by the following changes:
* pass argument `checkpoint=False` in https://github.com/BenevolentAI/DeeplyTough/blob/941bf23435617ce18ca40049ca46d32c8c2fbc09/deeplytough/engine/models.py#L69
* replace `nnf.sigmoid` with `torch.sigmoid` in https://github.com/BenevolentAI/DeeplyTough/blob/941bf23435617ce18ca40049ca46d32c8c2fbc09/deeplytough/engine/models.py#L61 and https://github.com/BenevolentAI/DeeplyTough/blob/941bf23435617ce18ca40049ca46d32c8c2fbc09/deeplytough/engine/models.py#L67

I will try to test it in the near future.
